test/load/packer/loadtest-ami/scripts/loadtest.js

```diff
@@ -14,49 +14,36 @@ export default function() {
   const kv_address = `${ipaddress + kv_uri + key}`
   
   //Put valid K/V
-  let res = http.put(kv_address, JSON.stringify(value));
+  let kvres = http.put(kv_address, JSON.stringify(value));
   if (
-    !check(res, {
-      'kv status code MUST be 200': (res) => res.status == 200,
+    !check(kvres, {
+      'kv status code MUST be 200': (kvres) => kvres.status == 200,
     })
   ) {
-    fail('kv status code was *not* 200');
+    fail(`registry check status code was *not* 200. error: ${kvres.error}. body: ${kvres.body}`)
   }
 
   //Register Service
-  data["ID"] = key;
   data["Name"] = key;
   const service_uri = '/v1/agent/service/register';
-  const service_address = `${ipaddress + service_uri }`
-  let res = http.put(service_address, JSON.stringify(data))
-  if (
-    !check(res, {
-      'register service status code MUST be 200': (res) => res.status == 200,
-    })
-  ) {
-    fail('register service status code was *not* 200');
-  }
 
-  //Register Check
-  check["ServiceID"] = key;
-  const check_uri = '/v1/agent/check/register';
-  const check_address = `${ipaddress + check_uri }`
-  let res = http.put(check_address, JSON.stringify(check))
+  const service_address = `${ipaddress + service_uri }`
+  let servres = http.put(service_address, JSON.stringify(data));
   if (
-    !check(res, {
-      'register check status code MUST be 200': (res) => res.status == 200,
+    !check(servres, {
+      'register service status code MUST be 200': (servres) => servres.status == 200,
     })
   ) {
-    fail('register check status code was *not* 200');
+    fail(`registry check status code was *not* 200. error: ${servres.error}. body: ${servres.body}`)
   }
-
 }
 
 export let options = {
-  // 1 virtual user
-  vus: 100,
-  // 1 minute
-  duration: "15m",
-  // 95% of requests must complete below 0.280s
-  thresholds: { http_req_duration: ["p(95)<280"] },
+  // 10 virtual user
```

Comment:
Minor correction for the comment to match the configuration:

```suggestion
  // 25 virtual users
```